### PR TITLE
Added timeOut Option

### PR DESCRIPTION
**Pull request by mattimatti**
Hello I've added a timeOut option as you suggested in #35. 
Thank you!

### CODE REVIEW

**Comment by stefanjudis** [comment]
Thanks will have a quick look soon. :)

**Comment by stefanjudis** [comment]
@mattimatti Test suite is failing. Can you have a look?

**Comment by mattimatti** [comment]
hello Stefan I've updated the test case. Introducing the hashbang handling I've changed the URL#QUALITY format to URL@QUALITY format. I assume it doesn't create problems out of the box. as this url naming is supposed to be internal. Feel free to reject and discuss better solution.

File: Gruntfile.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ module.exports = function(grunt) {
       waisenkinder : {
         options : {
           indexPath      : 'photobox/',
+          timeOut      : 1000,
           screenSizes    : [ '960', '350', '1200' ],
```
Comment (stefanjudis) [inline Gruntfile.js]: can we align this one? :blush:

**Comment by stefanjudis** [comment]
@mattimatti 

Hey hey,

thanks for putting the effort into.

I checkout out the PR and it doesn't seem to work. UI is broken and the generated images seem to have wrong file names.

screenshot 2015-04-13 20 38 49
screenshot 2015-04-13 20 39 17

**Comment by stefanjudis** [comment]
@mattimatti 

Any news on this one?

**Comment by stefanjudis** [comment]
@mattimatti 

Ping again. :)

I think I'm gonna close this one soon'ish, if no there is no response anymore.

**Comment by sbmadhav** [comment]
I think I can contribute to this. I'll need this anyway.

**Comment by stefanjudis** [comment]
If you're up for it I also can give you access. I'm not using this project anymore and would love to have someone bringing it further. :)

**Comment by sbmadhav** [comment]
Sad that you are not using it anymore. Its a great peice of work. Some more tweaks and it should be able to compete with the likes of huxley/ phantom css or wraith. Sure, I can own it.

**Pull request closed without merging**